src/reducers/url-state.js

```diff
@@ -88,6 +88,10 @@ const committedRanges: Reducer<StartEndRange[]> = (state = [], action) => {
     }
     case 'POP_COMMITTED_RANGES':
       return state.slice(0, action.firstPoppedFilterIndex);
+    case 'FULL_TIME_RANGE_REMOVED':
```

Comment:
So for this file, I believe it's still missing test coverage, and I think there are some correctness issues around indexes. I propose that for this PR, these actions are removed, and then follow-up with another PR with this work. 

I think it would be best to dispatch a single action, with all of the information needed to perform it, rather than doing it at such a granular level. Generally "fat" actions are better than granular ones as they reduce the churn for updating the store, and contain all the information to perform a single change.

For instance, rather than `FULL_TIME_RANGE_REMOVED`, and `HIDDEN_GLOBAL_TRACKS_REMOVED`, use an action like so:

```
type PIIAction = {
  type: 'PROFILE_PII_SANITIZED',
  oldMarkerIndexToNew: Map<IndexIntoRawMarkerTable, IndexIntoRawMarkerTable>,
  oldTrackIndexToNew: Map<TrackIndex, TrackIndex>,
  oldThreadIndexToNew: Map<ThreadIndex, ThreadIndex>,
  hiddenTracks: Set<TrackIndex>,
  ...
}
```